save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/ResultsChecker.kt

```diff
@@ -39,25 +39,28 @@ class ResultsChecker(
         val missingWarnings = expectedWarnings - expectedWarningsMatchedWithActual
         val unexpectedWarnings = actualWarnings - actualMatchedWithExpectedWarnings
 
+        val missing = missingWarnings.size
+        val match = expectedWarningsMatchedWithActual.size
+
         return when (missingWarnings.isEmpty() to unexpectedWarnings.isEmpty()) {
-            false to true -> Fail(
+            false to true -> Triple(Fail(
                 "$MISSING $missingWarnings",
                 "$MISSING (${missingWarnings.size}). $MATCHED (${expectedWarningsMatchedWithActual.size})"
```

Comment:
`missing` and `match` coutners can be used everywhere here too

Reply:
Fixed